cfme/configure/settings.py

```diff
@@ -53,272 +50,329 @@ def __init__(self, description=None, scope=None, days=None, hours=None, timezone
+            view.save.click()
 
     @property
     def template_quad(self):
-        navigate_to(self, 'All')
-        return self.template_quad
+        view = navigate_to(self, 'Quadicons')
+        return view.template_quad.read()
 
     @template_quad.setter
     def template_quad(self, value):
-        navigate_to(self, 'All')
-        fill(self.quadicons_form.template_quad, value)
-        sel.click(form_buttons.save)
-
-    def check_image_exists(self):
-        name = Quadicon.get_first_quad_title()
-        quad = Quadicon(name, None)
-        return quad.check_for_single_quadrant_icon
+        view = navigate_to(self, 'Quadicons')
+        if view.template_quad.fill(value):
+            view.save.click()
 
     @property
     def cloud_provider_quad(self):
-        navigate_to(self, 'All')
-        return self.cloud_provider_quad
+        view = navigate_to(self, 'Quadicons')
+        return view.cloud_provider_quad.read()
 
     @cloud_provider_quad.setter
     def cloud_provider_quad(self, value):
-        navigate_to(self, 'All')
-        fill(self.quadicons_form.cloud_provider_quad, value)
-        sel.click(form_buttons.save)
+        view = navigate_to(self, 'Quadicons')
+        if view.cloud_provider_quad.fill(value):
+            view.save.click()
 
     @property
     def timezone(self):
-        navigate_to(self, 'All')
-        return self.display_form.time_zone.first_selected_option_text
+        view = navigate_to(self, 'Display')
+        return view.time_zone.read()
 
     @timezone.setter
     def timezone(self, value):
-        navigate_to(self, 'All')
-        fill(self.display_form.time_zone, str(value))
-        sel.click(form_buttons.save)
+        view = navigate_to(self, 'Display')
+        if view.time_zone.fill(value):
+            view.save.click()
 
 
 visual = Visual()
 
 
+class VisualItemForm(MySettingsView):
+    grid_view = BootstrapSelect("perpage_grid")
+    tile_view = BootstrapSelect("perpage_tile")
+    list_view = BootstrapSelect("perpage_list")
+    reports = BootstrapSelect("perpage_reports")
+    save = Button("Save")
+    reset = Button("Reset")
+
+
+class VisualStartPageForm(MySettingsView):
+    show_at_login = BootstrapSelect("start_page")
+    save = Button("Save")
+    reset = Button("Reset")
+
+
+class VisualQuadiconsForm(MySettingsView):
+    infra_provider = BootstrapSwitch("quadicons_ems")
+    cloud_provider_quad = BootstrapSwitch("quadicons_ems_cloud")
+    host_quad = BootstrapSwitch("quadicons_host")
+    datastore_quad = BootstrapSwitch("quadicons_storage")
+    vm_quad = BootstrapSwitch("quadicons_vm")
+    template_quad = BootstrapSwitch("quadicons_miq_template")
+    long_text = BootstrapSelect("quad_truncate")
+    save = Button("Save")
+    reset = Button("Reset")
+
+
+class VisualDisplayForm(MySettingsView):
+    chart_theme = BootstrapSelect("display_reporttheme")
+    time_zone = BootstrapSelect("display_timezone")
+    save = Button("Save")
+    reset = Button("Reset")
```

Comment:
**REQURIED**
All these views are in the same page. So they should be part of the one view. Create a view let's say `VisualTabForm` and put all these views above as nested ones. In that case you don't need to create extra navigation steps, only one will be enough. Please do it for other tabs as well.

Reply:
I can implement what you have asked for Visual but for Timeprofile it's not possible as the landing pages is different for different navigation class (according to me) and remaining DefaultView and Defaufilter just have one navigation so not required.